sagemaker-fundamentals/create-training-job/create_training_job_vpc.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Need to <code>import os</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>